src/current/v24.3/set-up-logical-data-replication.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Set Up Logical Data Replication
+summary: Follow a tutorial to set up logical data replication between two clusters.
+toc: true
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+In this tutorial, you will set up **logical data replication (LDR)** between two CockroachDB clusters. Both clusters are active and can serve traffic. You can apply the outlined steps to create a _unidirectional_ LDR stream from a source to a destination (cluster A to cluster B). 
+
+To create a _bidirectional_ LDR stream, you can complete the [optional step](#step-4-optional-set-up-bidirectional-ldr-streams) to start a second LDR stream that sends writes from the table on cluster B to the table on cluster A. In a bidirectional setup, one cluster operates as both a source and a destination in separate LDR jobs. 
```

Comment:
can we be consistent and either say "LDR jobs" or "LDR streams"? Personally I prefer stream. What do you think?

Reply:
Yes, I can take a look at this and be more strict. I think defaulting to `stream` is fine, but there will be some places with `job` when we're talking about viewing jobs etc. It also tracks with the syntax we have in place `create logical replication stream` + `show logical replication jobs`. 

I'll go through and clean up the different situations for these though.

Reply:
Either gone through and adjusted mostly to plain LDR. I've used "stream" as verb for the majority, though I think there are some instances where saying LDR stream just reads easier. However, since we need to talk about looking at and monitoring jobs, I have kept LDR job in quite a few places. Happy to keep adjusting this as we work through the doc set for LDR prior to release.